pkg/kubelet/status/status_manager.go

```diff
@@ -539,7 +561,7 @@ func checkContainerStateTransition(oldStatuses, newStatuses []v1.ContainerStatus
 // updateStatusInternal updates the internal status cache, and queues an update to the api server if
 // necessary. Returns whether an update was triggered.
 // This method IS NOT THREAD SAFE and must be called from a locked function.
-func (m *manager) updateStatusInternal(pod *v1.Pod, status v1.PodStatus, forceUpdate bool) bool {
+func (m *manager) updateStatusInternal(pod *v1.Pod, status v1.PodStatus, forceUpdate, isSyncTerminatedPod bool) bool {
```

Comment:
use `podIsTerminal` consistently across all calls of this and generateAPIPodStatus, since they have the same meaning.

Reply:
Done